RxSwift/Schedulers/MainScheduler.swift

```diff
@@ -44,6 +45,13 @@ public final class MainScheduler : SerialDispatchQueueScheduler {
         }
     }
 
+    /// In case this method is running on a background thread it will throw an exception.
+    public class func ensureRunningOnMainThread(errorMessage: String? = nil) {
+        if !Thread.isMainThread {
```

Comment:
I think a guard is nicer in this case
```suggestion
        guard Thread.isMainThread else { rxFatalError ... }
```

Reply:
Fixed